src/components/ui/ListItem.svelte

```diff
@@ -1,10 +1,16 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte';
+  import type { Action } from 'svelte/action';
```

Comment:
Related to my content above about tooltips, you can just import `tooltip` directly into this component from [utilities/tooltip](https://github.com/NASA-AMMOS/aerie-ui/blob/develop/src/utilities/tooltip.ts)